lib/jazzy/config.rb

```diff
@@ -618,8 +611,9 @@ def validate
           '`framework_root` or `umbrella_header` may be ignored.'
       end
 
-      if custom_modules_configured && module_name_configured
-        raise 'Usage of config and comand line'
+      if modules_configured && module_name_configured
+        raise 'Jazzy only allows the use of a single command for generating documentation.' \
+        'Using both module configuration and modules configuration together is not supported.'
```

Comment:
This could also be a warning that we'll add the single module given to the list of modules and just proceed.